igvc_utils/include/igvc_utils/NodeUtils.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef NODEUTILS_HPP
+#define NODEUTILS_HPP
+
+#include <ros/ros.h>
+#include <stdlib.h>
+
+namespace igvc
+{
+    template<class T>
+    void param(const ros::NodeHandle pNh, const std::string &param_name, T &param_val, const T &default_val)
```

Comment:
So I just realized that this will not tell you from what node you came from. We should probably add that.

Reply:
You're referring to the messages that error messages that we get if the param isn't found right?

Reply:
I've added pNh.getNamespace(), which should be enough information to find out which node called the function I think.